tests/unit/test_tools_archive.py

```diff
@@ -0,0 +1,171 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+"""Tests for webcompat.com archive tools.
+
+TODO: test that an issue can be on a different repo
+"""
+
+import datetime
+import json
+import logging
+import pathlib
+
+import pytest
+import requests
+from requests.exceptions import HTTPError
+
+from tools.archive import model
+
+
+def get_fixture(filename):
```

Comment:
This is a cool method -- we could re-use this in other places where we do manually file traversal and file IO.

Reply:
yes we have plenty of functions in the project for testing that we should group in a utils module.